tools/model_tools.R

```diff
@@ -0,0 +1,39 @@
+#' Interpolate missing (rodent) abundance data
+#' 
```

Comment:
Provide some detail here on how the interpolation is done or change the first line to something like "Replace missing data using linear interpolation"